### ISSUE TITLE
Clicking on Compression Preset doesn't close the drop down

### ISSUE THREAD

**Issue by slrgt** [codeforreal1/compressO#8]
After opening the Compression Preset menu by clicking on it, if you click it again it doesn't close. You can only close it by clicking on a random spot in the application. Same for the Extension drop down menu.

**Comment by niraj-khatiwada** [comment]
Yeah, this bug annoyed me as well. It's from the UI library the app is using. I opened the [issue](https://github.com/nextui-org/nextui/issues/3056) and they've said it has been fixed in version 2.4.0 which they literally shipped after I made the build 1.2.0 public. So this will be fixed in next CompressO version.

**Comment by niraj-khatiwada** [comment]
This has been fixed in v1.3.0
